db/payment_icons.yml

```diff
@@ -1065,4 +1065,7 @@
 -
   name: genoapay
   label: Genoapay
+-
+  name: addi
+  label: ADDI
```

Comment:
@jforeroaddi, it looks like the issue is this change removes `group` from the `genoapay` entry. Can we fix this?

Reply:
@larouxn Done